queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * from animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * from animals;
+COMMIT;
+SELECT * from animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT save_point;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO save_point;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
```

Comment:
- [ Required ] I think this query as currently constructed may not answer the question _How many animals have never tried to escape?_ Currently, it returns the number of animals that have tried to escape which is actually the opposite of what it should do. Please consider fixing this.

## Example code 
```SQL
 SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
```